api.planx.uk/lib/hasura/metadata/index.test.ts

```diff
@@ -1,32 +1,40 @@
 import { createScheduledEvent, RequiredScheduledEventArgs } from "./index.js";
-import Axios, { AxiosError } from "axios";
+import axios from "axios";
+import type { Mocked } from "vitest";
 
-jest.mock("axios", () => ({
-  ...jest.requireActual("axios"),
-  post: jest.fn(),
-}));
-const mockAxios = Axios as jest.Mocked<typeof Axios>;
+describe("Creation of scheduled event", () => {
+  vi.mock("axios", async (importOriginal) => {
+    const actualAxios = await importOriginal<typeof import("axios")>();
+    return {
+      default: {
```

Comment:
Vitest does not assume, like jest, that a returned object is the `default` export of given module, so we have to be explicit about it, as per [the docs](https://vitest.dev/guide/migration.html#module-mocks).